velox/common/memory/MemoryArbitrator.h

```diff
@@ -68,6 +68,41 @@ class MemoryArbitrator {
     /// during the memory arbitration.
     uint64_t memoryPoolTransferCapacity{128 << 20};
 
+    /// When growing capacity, the growth bytes will be adjusted in the
+    /// following way:
+    ///  - If 2 * current capacity is less than or equal to
+    ///    'fastExponentialGrowthCapacityLimit', growing goes through fast path
+    ///    by at least doubling the current capacity, when conditions allow (see
+    ///    below NOTE section).
+    ///  - If 2 * current capacity is greater than
+    ///    'fastExponentialGrowthCapacityLimit', growing goes through slow path
+    ///    by growing capacity by at least 'slowCapacityGrowPct' * current
+    ///    capacity, when conditions allow (see below NOTE section).
+    ///
+    /// NOTE: In the conditions when original requested growth bytes is larger
+    /// than the adjusted growth bytes or adjusted growth bytes breaches max
+    /// capacity limit, the adjusted growth bytes will not be respected.
+    ///
+    /// NOTE: Growth bytes adjustment will be disabled when both
```

Comment:
```
s/Growth/growth/

NOTE: capacity growth adjust is enabled if both 'fastExponentialGrowthCapacityLimit' and 'slowCapacityGrowPct' are set, otherwise it is disabled.
```

Reply:
Capital letter at the start of a sentence?